src/aco.cpp

```diff
@@ -196,10 +206,50 @@ namespace full_coverage_path_planner
                         }
                     }
 
+                    //move ant
                     visited[new_location.pos.y][new_location.pos.x] = eNodeVisited;
                     ant.move(new_location, true);
                     ant.visited_counter++;
                     goals = map_2_goals(visited, eNodeOpen);
+
+                    //move ant extra according to velocity
+                    int moved = 1;
+                    int nx = new_location.pos.x;
+                    int ny = new_location.pos.y;
+
```

Comment:
Up to this point, ant has moved following ACO standard protocol.

Following comments discuss the changes to velocity ACO.

First, store the NEW location which ant moved to.